src/plugins/intel_npu/include/openvino/runtime/intel_npu/properties.hpp

```diff
@@ -0,0 +1,90 @@
+//
+// Copyright (C) 2022 Intel Corporation.
+// SPDX-License-Identifier: Apache 2.0
+//
+
+#pragma once
+
+#include <openvino/runtime/properties.hpp>
```

Comment:
Inside OV repository for OV includes the:
```suggestion
#include "openvino/runtime/properties.hpp"
```
Some files use it like in "src/plugins/intel_npu/src/al/include/variable_state.hpp", consider unify it.

Reply:
Changed from <> to ""